examples/acceptFromIDs.js

```diff
@@ -0,0 +1,63 @@
+// Required Modules
+const SteamUser = require('steam-user'),
+const SteamTotp = require('steam-totp'),
+const TradeOfferManager = require('steam-tradeoffer-manager'),
+const SteamCommunity = require('steamcommunity');
+ 
+// Initiate Modules
+let client = new SteamUser();
+let community = new SteamCommunity(),
+let manager = new TradeOfferManager({
+    "steam": client,
+    "domain": "nothinghere",
+    "language": "en"
+});
+ 
+let config = {
+    "username": "",
+    "password": "",
+    "identitySecret": "",
+    "sharedSecret": "",
+    "acceptSteamIDS": [
+        "SteamID",
+        "SteamID",
+        "SteamID"
+    ]
+};
+ 
+// Start sign in process
+client.logOn({
+    "accountName": config.username,
+    "password": config.password,
+    "twoFactorCode": SteamTotp.generateAuthCode(config.sharedSecret)
+});
+ 
+// Events (user signed in, cookies retrieved, new offer retrieved)
+client.on('loggedOn', () => {
+    console.log(`Logged into Steam as ${client.steamID.getSteam3RenderedID()}`);
+    client.setPersona(SteamUser.EPersonaState.Online);
+});
+ 
+client.on('webSession', (sessionID, cookies) => {
+    manager.setCookies(cookies, err =>  {
+        if (err) {
+            console.error(err);
```

Comment:
First apperance of `console.error()` in the code. It's not bad (I guess), but `console.log()` was used in all other examples.